airbyte-integrations/bases/base-normalization/normalization/transform_catalog/transform.py

```diff
@@ -185,25 +185,32 @@ def strip_accents(s):
     return "".join(c for c in ud.normalize("NFD", s) if ud.category(c) != "Mn")
 
 
+# Temporarily disabling the behavior of the ExtendedNameTransformer, see (issue #1785)
 def normalize_identifier_name(input_name: str, integration_type: str) -> str:
-    # Temporarily disabling the behavior of the ExtendedNameTransformer, see (issue #1785)
-    # if integration_type == "bigquery":
+    # all tables (even quoted ones) are coerced to lowercase.
+    if integration_type == "redshift":
+        input_name = input_name.lower()
```

Comment:
it would be nice if we could somehow apply all these destination logic on top of `cleanedName` or `aliasName` fields inside the ConfiguredCatalog (for the destination at least)

So everything is already cleaned out properly with the right "dialect" when we arrive in normalization

Reply:
yup. someday when we "fix" our destination abstraction. :wink: